hack/update-deps.sh

```diff
@@ -20,4 +20,43 @@ set -o pipefail
 
 source $(dirname "$0")/../vendor/knative.dev/hack/library.sh
 
+version=$(echo $@ | grep -o "\-\-release \S*" | awk '{print $2}' || echo "")
```

Comment:
```
hack/update-deps.sh # optionally with --upgrade to upgrade submodules
```

can we document this?

Reply:
That's what you use on every Knative repository already, it is "extended" to Git submodules too, not sure if I would document something that common here